### PR TITLE
update

### PR DESCRIPTION
**Pull request by vamtic**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/www.vamtic.json
Comment (wdhdev) [inline domains/www.vamtic.json:1]: We don't allow `www` nested subdomains

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/www.vamtic.json
Comment (vamtic) [reply domains/www.vamtic.json:1]: why :c

File: domains/info.vamtic.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "vamtic",
+    "email": "vamtic@yahoo.com"
+  },
+  "record": {
+    "CNAME": "vamtic.vercel.app"
```
Comment (wdhdev) [inline domains/info.vamtic.json:7]: We don't support Vercel at the moment.

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/www.vamtic.json
Comment (wdhdev) [reply domains/www.vamtic.json:1]: I don't even know we just don't

File: domains/info.vamtic.json (hunk 1 above)
Comment (vamtic) [reply domains/info.vamtic.json:7]: R

**Review by wdhdev — APPROVED** [review]

**Pull request merged**